app/i18n/de.json

```diff
@@ -1,7 +1,9 @@
 {
   "ABONNER_BANNER": {
+    "BANNER_TEXT_LINE_1": "",
+    "BANNER_TEXT_LINE_2": "",
     "LINK_TEXT": "Gehe zu abonner.varsom.no",
-    "TITLE": "Varsom Abonner"
+    "TITLE": ""
```

Comment:
Når det står tom string for en del oversettelser, menes det at det skal brukes engelsk tekst for dette? Sjekken er om nøkkelen finnes, ikke om det er tom string, så nå tror jeg det kommer til å bli mye tomme tekster..